content/blogposts/blog-post-starter-pack.md

```diff
@@ -0,0 +1,255 @@
+| 1    | Iris Setosa | 5.1               | 3.5              | 1.4               | 0.2              |
+| 2    | Iris Setosa | 4.9               | 3.0              | 1.4               | 0.2              |
+</TableWrapper>
+
+This is the same table, but written in **HTML**:
+
+<TableWrapper>
+  <table>
+    <thead>
+      <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Sepal Length (cm)</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>1</td>
+        <td>Iris Setosa</td>
+        <td>5.1</td>
+        <td>3.5</td>
+        <td>1.4</td>
+        <td>0.2</td>
+      </tr>
+      <tr>
+        <td>2</td>
+        <td>Iris Setosa</td>
+        <td>4.9</td>
+        <td>3.0</td>
+        <td>1.4</td>
+        <td>0.2</td>
+      </tr>
+    </tbody>
+  </table>
+</TableWrapper>
+
+
+### YouTube
+
+You can easily embed YouTube videos using the `YouTube` component. The `id` value is found at the end of the URL for any YouTube video. To display the title below the video, add the `showTitle` property.
+
+<YouTube
+  title="Accelerate developer onboarding with Sourcegraph" 
+  id="DgwvhRW1Cbc"
+  showTitle={true}
+/>
+
+
+### Video
+
+The `Video` component supports .mp4 and .webm files. First, upload the two versions of the video to Google Cloud. Then, add the root names of these files to the `source` property, without including their extensions. You can choose to repeat a video by including the `loop` property, or display the caption using the `showCaption` property.
+
+<Video 
+  source={{webm: 'blog/shift-left/01-code-reuse', mp4: 'blog/shift-left/01-code-reuse'}} 
+  loop={true}
+  title="Code reuse"
+  caption="An example of code search with Sourcegraph"
+  showCaption={true}
+/>
+
+
+### HubSpot Form Example
+
+Use the `EmbeddedHubSpot` component to drop a custom HubSpot form into your post. In most cases, the `formId` is the only property that needs to be updated. This is the same `formId` dictated by the HubSpot console. Once added to your post, the HubSpot form will render wherever you place the `div` using the specified `id` value.
+
+<EmbeddedHubSpot 
+  portalId='2762526' 
+  formId='08e6c442-0e7c-4892-a262-76dae55ab497' 
+  targetId='#sign-up-form' />
+
+<div id="sign-up-form"></div>
+
+
+## Code
+
+Any code you write, you can style! The Blog uses Prism for code formatting. Specify the language at the front of the code block. 
+
+**TypeScript**
+
+```ts
+export const BlogPost<BlogPostProps> = ({title, text}) = {
+  <h1>{title}</h1>
+  <p>{text}</p>
+  …
+}
+```
+
+
+## Links
+
+Add links with an anchor tag as you usually would: 
+
+<div className="text-center mb-5">
+  <a href="https://www.sourcegraph.com/search">Check out Sourcegraph Search!</a>
+</div>
+
+If you are wrapping an image in a link, please include a `title` on the anchor tag for accessibility.
+
+<a href="https://www.sourcegraph.com/search" title="Go to Sourcegraph Code Search">
```

Comment:
We should try to make the titles also describe the image along with the link since the screen reader will skip over the image when the code is written this way. It may take some trial and error to see what the screen reader says & what makes sense.

Reply:
Great point! What do you think of adding this context to the description, then altering the link title and image alt text? Maybe something like this:

```suggestion
If you are wrapping an image in a link, please include a `title` on the anchor tag for accessibility, this way screen readers can pick up all the content. Screen readers may skip over the image, so it is important for the `title` to include a description of the image and intended action.

<a href="https://www.sourcegraph.com/search" title="An image of the Sourcegraph logo that links to Sourcegraph Code Search">
  <Figure 
    src="https://storage.googleapis.com/sourcegraph-assets/blog/default_hero_social.png"
    alt="Sourcegraph logo thumbnail"
  />
</a>
```

Reply:
So the screen reader will read the link text & the image alt text only if the anchor tag surrounds a pure img tag, but when it surrounds the our Figure component it doesn't. Have you tested it? You can test with this blog `/blog/optimizing-a-code-intel-backend`. I think we should reconfigure the `Figure` component to make it more accessible, or encourage blog writers to not make complex images a link. Or instruct if the image is also a link, just to use the pure `<img/ >` tag instead of Figure. What do you think?

Reply:
I agree! I don't think we need to link Figures or images even @st0nebraker but maybe include a link the caption/cite if need be.

Reply:
Update made [8889b16](https://github.com/sourcegraph/about/pull/5553/commits/8889b16002511333223064eaba8e642c711bf206) :)